db/post_migrate/20250526063633_copy_add_groups_to_about_component_settings.rb

```diff
@@ -9,15 +9,19 @@ class CopyAddGroupsToAboutComponentSettings < ActiveRecord::Migration[7.2]
   }
 
   def up
+    theme_id = DB.query_single(<<~SQL).first
+      SELECT id
+      FROM themes
+      WHERE name = 'Add Groups to About'
+      AND enabled = true
+    SQL
```

Comment:
The previous migration was blowing up because of `ERROR:  more than one row returned by a subquery used as an expression`. Do we ignore the other theme that is enabled and has the same name?